### DISCUSSION TITLE
AppImage build

### DISCUSSION THREAD

**Discussion by AsciiWolf** [hmatuschek/qdmr#364] [General]
Please, consider making an AppImage (or a Flatpak) build of qdmr for Linux users. qdmr is still not available (or is outdated) in many distributions and having to compile it from source is not ideal.

**Comment by allesand** [comment]
There is a snap available, is that a usable format? According to https://repology.org/project/qdmr/versions most distros are quite up to date, which distro did you find with an outdated version? Snap format has some issues with USB access, though.

**Comment by AsciiWolf** [comment]
Snap is mostly Ubuntu-specific - basically nobody uses it outside of Ubuntu. Flatpak or AppImage work (if done correctly) on every distro + there are special immutable distributions like Fedora Silverblue, Vanilla OS, Endless OS and others that do not support regular deb/rpm packages.

**Comment by hmatuschek** [comment]
I would not maintain further distributional packages, I have several people that maintain those for different distributions. Of cause, feel free to adopt it and provide a Flatpak for all to use.
